src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -368,23 +467,77 @@ def _parse_lttng_data(
             begin = events[0][LttngEventFilter.TIMESTAMP]
             end = events[-1][LttngEventFilter.TIMESTAMP]
 
+            init_events = []
+            run_events = []
+
+            # 全てのトレースイベントを init_events, run_eventsに振り分ける
+            init_event_names = set(Lttng._prioritized_init_events)
             for event in events:
+                event_name = event[LttngEventFilter.NAME]
+                if event_name in init_event_names:
+                    init_events.append(event)
+                else:   # MYK test
+                    run_events.append(event)
+
+            # init_events をソートする
+            import functools
+            init_events.sort(key=functools.cmp_to_key(Lttng._compare_init_event))
+            # 初期化系トレースイベントを処理する
+            handler.create_init_handler_map()
+            for event in init_events:
+                event_name = event[LttngEventFilter.NAME]
+                if event_name in handler.handler_map:
+                    handler_ = handler.handler_map[event_name]
+                    handler_(event)
```

Comment:
I suggest to remove if statement because it should be always true.

```suggestion
                handler_ = handler.handler_map[event_name]
                handler_(event)
```

Reply:
I removed the if statement